masterlist.yaml

```diff
@@ -742,113 +818,178 @@ plugins:
-    url: ['https://www.nexusmods.com/skyrimspecialedition/mods/2259/']
+  - name: 'JK''s Whiterun.esp'
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/2259/'
+        name: 'JK''s Whiterun on Nexus Mods'
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
     tag:
       - C.Location
       - C.Water
   - name: 'JKs Whiterun exterior.esp'
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/2259/'
+        name: 'JK''s Whiterun on Nexus Mods'
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
   - name: 'Bring Out Your Dead.esp'
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/323/'
+        name: 'Bring Out Your Dead on Nexus Mods'
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
     global_priority: -90
   - name: 'Darkwater Crossing.esp'
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/326/'
+        name: 'Darkwater Crossing on Nexus Mods'
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
       - 'Immersive Citizens - AI Overhaul.esp'
   - name: 'Helarchen Creek.esp'
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/4043'
+        name: 'Helarchen Creek on Nexus Mods'
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
   - name: 'Ivarstead.esp'
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/349'
+        name: 'Ivarstead on Nexus Mods'
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
   - name: 'Karthwasten.esp'
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/350/'
+        name: 'Karthwasten on Nexus Mods'
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
   - name: 'Kynesgrove.esp'
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/351/'
+        name: 'Kynesgrove on Nexus Mods'
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
   - name: 'Shor''s Stone.esp'
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/354/'
+        name: 'Shor''s Stone on Nexus Mods'
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
   - name: 'Soljund''s Sinkhole.esp'
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/358/'
+        name: 'Soljund''s Sinkhole on Nexus Mods'
     after:
       - 'Verdant - A Skyrim Grass Plugin SSE Version.esp'
   - name: 'Whistling Mine.esp'
+    url: 
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/367/'
+        name: 'Whistling Mine on Nexus Mods'
+      - link: 'https://afkmods.iguanadons.net/index.php?/files/file/1912-whistling-mine/'
+        name: 'Whistling Mine on AFK Mods'
+      - link: 'https://mods.bethesda.net/#en/workshop/skyrim/mod-detail/2959645/'
+        name: 'Whistling Mine on Bethesda.net'
```

Comment:
Thought on the other links for afk & beth, are they unnecessary?

Reply:
No, `url` is a list partly to allow for cases where a mod may be hosted multiple places - if one goes down, it can be useful to have another link handy.